doc/lua_api.txt

```diff
@@ -2752,6 +2756,14 @@ and `minetest.auth_reload` call the authetification handler.
     * spread these updates to neighbours and can cause a cascade
       of nodes to fall.
 
+### Mod channels
+You can find mod channels communication scheme in `docs/mod channels.png`.
+
+* `minetest.mod_channel_join(channel_name)`
```

Comment:
`mod_` is redundant.

Reply:
not exactly, if we implement coreside (for example) chat channels for MMO minetest you loose the precision